tiup/tiup-component-cluster-scale-in.md

```diff
@@ -0,0 +1,70 @@
+---
+title: tiup cluster scale-in
+---
+
+# tiup cluster scale-in
+
+The `tiup cluster scale-in` command is used to scale in the cluster, which takes the services of the specified nodes offline, removes the specified nodes from the cluster, and deletes the remaining files from those nodes.
+
+Because the TiKV, TiFlash, and TiDB Binlog components are taken offline asynchronously (which requires TiUP to remove the node through API first) and the stopping process takes a long time (which requires TiUP to continuously check whether the node is successfully taken offline), the TiKV, TiFlash, and TiDB Binlog components are handled particularly as follows:
+
+- For TiKV, TiFlash and TiDB Binlog components:
+
+    1. TiUP takes the node offline through API and directly exits without waiting for the process to be completed.
+    2. To check the status of the nodes being scaled in, you need to execute the `tiup cluster display` command, and wait for the status to be changed to `Tombstone`.
+    3. To clean up the nodes in the `Tombstone` status, you need to execute the `tiup cluster prune` command. The `tiup cluster prune` command performs the following operations:
+
+        - Stops the services of the nodes that have been taken offline.
+        - Cleans up the data files of the nodes that have been taken offline.
+        - Updates the cluster topology and remove the nodes that have been taken offline.
+
+- For other components:
+
+    - When taking the PD components offline, TiUP cluster quickly deletes the specified nodes from the cluster through API, stops the service of the specified PD nodes, and then deletes the related data files from the nodes.
+    - When taking other components down, TiUP cluster directly stops the node services and deletes the related data files from the specified nodes.
```

Comment:
- This part should not be under "the TiKV, TiFlash, and TiDB Binlog components are handled particularly as follows:".
- "TiUP cluster"  is very likely to be understood as a cluster, not a component. Is "TiUP Cluster" better able to indicate that it's a component of TiUP? @lucklove 
```suggestion
For other components:

- When taking the PD components offline, TiUP Cluster quickly deletes the specified nodes from the cluster through API, stops the service of the specified PD nodes, and then deletes the related data files from the nodes.
- When taking other components down, TiUP Cluster directly stops the node services and deletes the related data files from the specified nodes.
```